apps/i18n/common/en_us.json

```diff
@@ -2869,6 +2869,13 @@
   "progressV2_feedback_shareMoreLinkText": "Share more feedback",
   "progressV2_feedback_thumbsUp": "I like it!",
   "progressV2_feedback_thumbsDown": "I don't like it",
+  "submitProjectGallery_alreadySubmitted": "Submitted to Featured Projects gallery",
+  "submitProjectGallery_details": "*Submitting a project does not guarantee that it will be featured. Once submitted, this project cannot be submitted again.",
```

Comment:
I find it more legible to have a space after the `*`.

Reply:
@samantha-code - lmk what you think about Brendan's question and suggestion above. Thanks!

Reply:
Im fine with the simplification brendan is suggesting

Reply:
rereading the suggestion! There is no * above 

Maybe we just have the bottom saying "Submitting a project does not guarantee that it will be featured. The project can only be submitted once." without the *